app/controllers/api/v0/articles_controller.rb

```diff
@@ -72,11 +72,18 @@ def update
 
       def article_params
         allowed_params = [
-          :title, :body_markdown, :published, :series, :publish_under_org,
+          :title, :body_markdown, :published, :series,
           :main_image, :canonical_url, :description, tags: []
         ]
+        allowed_params << :organization_id if params["article"]["organization_id"] && allowed_to_change_org_id?
```

Comment:
@rhymes @Glennmen I think this should work for now; thoughts?

I'll add a test for it soon. 🙃